### PR TITLE
Removed all references to Pimple's register method

### PR DESCRIPTION
**Pull request by silentworks**

### CODE REVIEW

**Comment by silentworks** [comment]
Since this has been changed in all the other repo's, can we get this merged in please. //cc @codeguy

File: docs/features/templates.md [hunk 1]
```diff
@@ -85,9 +85,11 @@ $app = new \Slim\App();
...
 
 // Register Twig View service
-$container->register(new \Slim\Views\Twig('path/to/templates', [
-    'cache' => 'path/to/cache'
-]));
+$container['view'] = function () {
+    return new \Slim\Views\Twig('path/to/templates', [
+        'cache' => 'path/to/cache'
+    ]);
+};
```
Comment (akrabat) [inline docs/features/templates.md:100]: This should match the Twig-View README and add the extension too.

Comment (silentworks) [inline docs/features/templates.md:100]: I just followed the existing doc, which didn't mention TwigExtension in it. Should I still change this?

Comment (akrabat) [inline docs/features/templates.md:100]: Yes.

To expand on this. When this doc was written, the extension didn't exist and the helper was part of the main class.

**Comment by codeguy** [comment]
Thanks!

**Pull request merged**